erpnext/controllers/accounts_controller.py

```diff
@@ -594,6 +596,21 @@ def update_against_document_in_jv(self):
 			from erpnext.accounts.utils import reconcile_against_document
 			reconcile_against_document(lst)
 
+	def set_pricing_rules_table(self):
+		if self.doctype in ['Purchase Order', 'Sales Order', 'Sales Invoice', 'Purchase Invoice',
+							'Supplier Quotation', 'Purchase Receipt', 'Delivery Note', 'Quotation']:
+			# Clear the child table if already set
+			# Only pricing rules which are applied will show up here
+			self.set("pricing_rules", [])
+
+			for item in self.items:
+				if item.pricing_rules:
+					self.append("pricing_rules", {
+						"pricing_rule": item.pricing_rules.split(",")[0], 
```

Comment:
Why are we showing only one pricing rule? If multiple rules applied, we should row all rules in different rows.